content/kubernetes/release-notes/k8s-6-4-2-3.md

```diff
@@ -34,7 +34,19 @@ The key features, bug fixes, and known limitations are described below.
 
 ## Upgrade to 6.4.2-3
 
-This release uses a new file for the `ValidatingWebhookConfiguration`. To use the 6.4.2-3 release, delete the old file `redb-admission.yaml` and apply the new file `webhook.yaml`.
+This release uses a new file for the `ValidatingWebhookConfiguration`. To use the 6.4.2-3 release, delete the old file and apply the new file.
```

Comment:
Maybe phrase this around webhook resources and not around files, something along these lines:

```suggestion
This release uses a new `ValidatingWebhookConfiguration` resource that replaces the old webhook resource. To use the 6.4.2-3 release, delete the old webhook resource and apply the new file.
```